lib/knapsack_pro/hooks/queue.rb

```diff
@@ -3,13 +3,18 @@ module Hooks
     class Queue
       class << self
         attr_reader :before_queue_store,
+          :before_subset_queue_store,
           :after_subset_queue_store,
           :after_queue_store
 
         def reset_before_queue
           @before_queue_store = nil
         end
 
+        def reset_before_subset_queue
+          @before_subset_queue_store = nil
```

Comment:
I noticed specs where flaky (randomly failing) and I could not figure out why. It turned out there was a wrong name for this instance variable. I've fixed it and now specs are passing.